add-ons/istio/README.adoc

```diff
@@ -0,0 +1,52 @@
+:linkattrs:
+
+= Istio Add-on
+
+An addon that will deploy https://Istio.io/[istio] service mesh on to Openshift.
+
+Verify you have installed these addons, by following the link:../../README.adoc#download-and-use-community-add-ons[general readme]
+
+== minishift profile setup
+
+As https://Istio.io/[istio] setup spins up multiple pods its ideal to have a setup with the following configuration
+
+[code,sh]
+----
+minishift profile set servicemesh <1>
```

Comment:
I think double hash `#` is needed here and in following lines as well. See http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#source-code

Reply:
`##` for what ?

Reply:
That's seems to be syntax. Chec `Code block with non-selectable callouts` here http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#source-code

```
minishift profile set servicemesh # <1>
```